docker/build/installers/install_qt.sh

```diff
@@ -51,8 +51,8 @@ cuteci \
     --packages "qt.qt5.${QT_VERSION_Z}.gcc_64" \
     --keep-tools
 
-ln -s ${MY_DEST_DIR}/${QT_VERSION_B}/gcc_64/include /usr/local/include/qt5
-ln -s ${MY_DEST_DIR}/${QT_VERSION_B}/gcc_64/lib /usr/local/lib/qt5
+# Make qt5 version agnostic to end users
```

Comment:
Can we just say "Hide qt5 version from end users"? "Agnostic" is related to religion.

Reply:
> Can we just say "Hide qt5 version from end users"? "Agnostic" is related to religion.

Suggestion accepted.  Got this word from some tech books originally.